dev-python/reportlab/files/reportlab-3.6.11-correct-srclen-type-in-gstate__aapixbuf.patch

```diff
@@ -0,0 +1,32 @@
+From 9ba2380a68e891667a2659e16d0f04ebe0ecfd19 Mon Sep 17 00:00:00 2001
```

Comment:
Include ink to the upstream commit here too please

Reply:
There isn't one this time.  They don't do development over git, it's in SVN and it's not open for patches.  I tried to submit to their -users mailing list but it rejected my mail server.

Reply:
Oh bleh. Okay. Sorry, I remember now, tried to submit something ages ago.

Can you add a comment to that effect at the top? We should ask someone to forward it - @mgorny was able to get through in the past (I couldn't).

Reply:
Done